src/shared-components/BaseModal/BaseModal.spec.tsx

```
@@ -1,6 +1,6 @@
 import React from 'react';
 import { describe, it, expect, vi, afterEach } from 'vitest';
-import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import { render, screen, waitFor, fireEvent, cleanup } from '@testing-library/react';
```

ESLint

[error] Replace `·render,·screen,·waitFor,·fireEvent,·cleanup·` with `⏎··render,⏎··screen,⏎··waitFor,⏎··fireEvent,⏎··cleanup,⏎`

(prettier/prettier)